<ui version="4.0" >
 <class>crystalConfig</class>
 <widget class="QWidget" name="crystalConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure Pager</string>
  </property>
  <property name="accessibleName" >
   <string>Configure Pager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="1" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" >
    <spacer name="verticalSpacer_4" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>47</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <widget class="QGroupBox" name="appearenceGroupBox" >
     <property name="title" >
      <string/>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="displayLabel" >
        <property name="text" >
         <string>Maximum matches</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="matchesSpin" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Change the number of rows</string>
        </property>
        <property name="whatsThis" >
         <string>Change the number of rows</string>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="labelRows_3" >
        <property name="text" >
         <string>Show folders</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>timeoutSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QCheckBox" name="showFoldersCheck" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Timeout in seconds</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="timeoutSpin" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Change the number of rows</string>
        </property>
        <property name="whatsThis" >
         <string>Change the number of rows</string>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Icon size</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QSlider" name="sizeSlider" >
        <property name="maximum" >
         <number>6</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="labelRows" >
        <property name="text" >
         <string>Default query</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>timeoutSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QCheckBox" name="clipboard" >
        <property name="text" >
         <string>use clipboard</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QLineEdit" name="defaultQuery" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="configButton" >
       <property name="text" >
        <string>Configure Desktop Search...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>333</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>configButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
